users/kuatsure/kuatsure.c

```diff
@@ -30,7 +31,9 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
     case KB_FLSH:
       if (!record->event.pressed) {
         SEND_STRING("make " QMK_KEYBOARD ":" QMK_KEYMAP
-          #if  (defined(BOOTLOADER_DFU) || defined(BOOTLOADER_LUFA_DFU) || defined(BOOTLOADER_QMK_DFU))
+          #if defined(__arm__)
```

Comment:
Instead, you may want to change all of these to `:flash`, as this handles this, a bit better.